sos/report/plugins/anaconda.py

```diff
@@ -24,21 +24,21 @@ class Anaconda(Plugin, RedHatPlugin):
 
     def setup(self):
 
-        paths = [
+        self.copypaths = [
             "/root/anaconda-ks.cfg"
         ]
 
         if self.path_isdir('/var/log/anaconda'):
             # new anaconda
-            paths.append('/var/log/anaconda')
+            self.copypaths.append('/var/log/anaconda')
         else:
-            paths = paths + [
+            self.copypaths = self.copypaths + [
                 "/var/log/anaconda.*",
                 "/root/install.log",
                 "/root/install.log.syslog"
             ]
 
-        self.add_copy_spec(paths)
+        self.add_copy_spec(self.copypaths)
```

Comment:
The credentials can be in either anaconda log or in the `*ks.cfg` file - hence apply postproc to all copyspecs.